### PR TITLE
Initial draft of financial documentation

### PR DESCRIPTION
**Pull request by kberzinch**
- Run Linkcheck during build process
- Start calendar of tasks: https://d3eefxa5h2bcis.cloudfront.net/corporate/calendar/
- Start bookkeeping standards: https://d3eefxa5h2bcis.cloudfront.net/corporate/finance/bookkeeping-standards/
- Start documenting monthly reconciliation processing: https://d3eefxa5h2bcis.cloudfront.net/corporate/finance/monthly-reconciliation/

### CODE REVIEW

**Review by JoshOldenburg — DISMISSED** [review]
Looks good overall. One small suggestion.

File: corporate/calendar.rst [hunk 1]
```diff
@@ -1,27 +1,90 @@
...
 .. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+This page describes tasks that must be completed on a set schedule.
+
+January
+-------
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Submit Sales & Use Tax Return to the Georgia Department of Revenue
```
Comment (knxtdoor) [inline corporate/calendar.rst:18]: Where/how does this happen?

File: corporate/calendar.rst [hunk 2]
```diff
@@ -1,27 +1,90 @@
...
+* `File Annual Registration with the Georgia Secretary of State <https://sos.ga.gov/how-to-guide/how-file-annual-registration>`_
+* Hold quarterly board meeting
```
Comment (knxtdoor) [inline corporate/calendar.rst:20]: Will there be a board meeting planning page?

File: corporate/calendar.rst [hunk 3]
```diff
@@ -1,27 +1,90 @@
...
+* Reconcile 1099-K received from Square to QuickBooks records
```
Comment (knxtdoor) [inline corporate/calendar.rst:21]: Link to 1099-K purpose/description

File: corporate/calendar.rst [hunk 4]
```diff
@@ -1,27 +1,90 @@
...
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Begin compiling documentation for Internal Revenue Service Form 990-EZ
+* Hold quarterly board meeting - elect directors and officers
+
+August
+------
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+* Audit BILL Spend & Expense cardholders
```
Comment (knxtdoor) [inline corporate/calendar.rst:67]: Mention collaboration with student officers?

File: corporate/calendar.rst [hunk 5]
```diff
@@ -1,27 +1,90 @@
...
+* Renew Northwest Registered Agent contract
+
+June
+----
+
+* :doc:`Complete monthly financial reconciliation and reporting </corporate/finance/monthly-reconciliation>`
+
+July
+----
+
```
Comment (knxtdoor) [inline corporate/calendar.rst:56]: Mention start of GT fiscal year

File: corporate/finance/bookkeeping-standards.rst [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+.. vale Google.Parens = NO
+.. vale Google.Passive = NO
+.. vale Google.Will = NO
+.. vale proselint.Typography = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+.. vale write-good.TooWordy = NO
+
+RoboJackets financial records should comply with Generally Accepted Accounting Principles and Internal Revenue Service standards by default.
```
Comment (knxtdoor) [inline corporate/finance/bookkeeping-standards.rst:14]: Link to GAAP standards

File: corporate/finance/bookkeeping-standards.rst [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+This page documents standards for specific transactions.
+
+Restricted funds
+----------------
+
+RoboJackets, Inc. generally doesn't accept donations.
+As a 501(c)(7) exempt organization, donations aren't tax-deductible for donors.
+Most donors should be directed to Georgia Tech Foundation, Inc. or another Georgia Tech affiliate to process donations.
```
Comment (knxtdoor) [inline corporate/finance/bookkeeping-standards.rst:22]: "If someone wants to donate to RoboJackets, they should be directed to the Georgia Tech Foundation"

File: corporate/finance/monthly-reconciliation.rst [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+RoboJackets responsibilities
+----------------------------
+
+These tasks should be completed by end of day on the first day of the following month.
+
+* `Sync BILL Spend & Expense transactions to QuickBooks <https://help.bill.com/direct/s/article/6854501>`_
+* Sync `invoices <https://loop.robojackets.org/docs/quickbooks/invoices/>`_ and `payments <https://loop.robojackets.org/docs/quickbooks/payments/>`_ from Loop to QuickBooks
```
Comment (knxtdoor) [inline corporate/finance/monthly-reconciliation.rst:19]: Workday sync mention? or maybe in loop docs already

File: corporate/finance/bookkeeping-standards.rst (hunk 7 above)
Comment (kberzinch) [reply corporate/finance/bookkeeping-standards.rst:22]: Can you add a suggestion in GitHub, I'm not sure what you're getting at here

File: corporate/finance/monthly-reconciliation.rst (hunk 8 above)
Comment (kberzinch) [reply corporate/finance/monthly-reconciliation.rst:19]: It is covered in the Loop documentation here: https://loop.robojackets.org/docs/workday/data-load/

File: corporate/calendar.rst (hunk 1 above)
Comment (kberzinch) [reply corporate/calendar.rst:18]: It is stubbed out here: https://github.com/RoboJackets/business-process-manual/blob/main/corporate/merchandise-sales.rst

I'll come back to it later

Comment (kberzinch) [reply corporate/calendar.rst]: Added separate stub page for sales tax

File: corporate/calendar.rst (hunk 2 above)
Comment (kberzinch) [reply corporate/calendar.rst]: Added stub page

File: corporate/calendar.rst (hunk 3 above)
Comment (kberzinch) [reply corporate/calendar.rst]: Added stub page

File: corporate/calendar.rst (hunk 5 above)
Comment (kberzinch) [reply corporate/calendar.rst:56]: That's not a task

File: corporate/calendar.rst (hunk 4 above)
Comment (kberzinch) [reply corporate/calendar.rst]: Added link to stub page

File: corporate/finance/bookkeeping-standards.rst (hunk 6 above)
Comment (kberzinch) [reply corporate/finance/bookkeeping-standards.rst]: Added link to FASB ASC

File: corporate/finance/bookkeeping-standards.rst (hunk 7 above)
Comment (knxtdoor) [reply corporate/finance/bookkeeping-standards.rst:22]: I think "If someone wants to donate to RoboJackets, they should be directed to the Georgia Tech Foundation" is a better phrasing than "Most donors should be directed to Georgia Tech Foundation, Inc. or another Georgia Tech affiliate to process donations."

Comment (kberzinch) [reply corporate/finance/bookkeeping-standards.rst]: Pushed a different phrasing

**Review by knxtdoor — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**